docs/source/developer_resources.rst

```diff
@@ -426,21 +410,50 @@ Older versions of the source code documentation are (still) on readthedocs; howe
 
 For releasing, copy the ``htmlout`` directory into the seed-platform's website repository under ``docs/code_documentation/<new_version>``. Make sure to add the new documentation to the table in the ``docs/developer_resources.md``.
 
+Contribution Instructions / Best Practices
+------------------------------------------
+
+If this is the first time contributing and you are outside of the DOE National Lab system, then you will need to review and fill out the contribution agreement which is found in `SEED's Contribution Agreement in the GitHub repository`_
+
+The desired workflow for development and submitting changes is the following:
+
+#. Fork the repository on GitHub if you do not have access to the repository, otherwise, work within the https://github.com/seed-platform/seed repository.
+#. Ensure there is a ticket/issue created for the work you are doing. Verify that the ticket is assigned to you and that it is part of the latest project board on the GitHub site (https://github.com/orgs/SEED-platform/projects).
+#. Move the ticket/issue to 'In Progress' in the GitHub project tracker when you begin work
+#. Create a branch off of develop (unless it is a hotfix, then branch of the appropriate tag). The recommended naming convention is <issue_id>-short-descriptive-name.
+#. Make changes and write a test for the code added.
+#. Make sure tests pass locally. Most branches created and pushed to GitHub will be tested automatically.
+#. Upon completion of the work, create a pull request (PR) against the develop branch (or hotfix branch if applicable). In the PR description fill out the requested information and include the issue number (e.g., #1234).
+#. Assign one label to the PR (not the ticket/issue) in order to auto-populate change logs (e.g., Bug, Feature, Maintenance, Performance, DoNotPublish) This is required and CI will fail if not present.
```

Comment:
What is "CI" here?

Reply:
CI = continuous integration. It is what runs the software tests on GitHub